src/govuk/settings/_compatibility.scss

```diff
@@ -56,16 +72,26 @@ $govuk-compatibility-govuktemplate: false !default;
 ///
 /// @type Boolean
 /// @access public
+/// @deprecated Will be removed in v5.0 with the rest of the compatibility mode
+/// suite of tools and settings
 
 $govuk-compatibility-govukelements: false !default;
 
+@if $govuk-compatibility-govukelements == true {
+  @include _warning("compatibility-mode", "$govuk-compatibility-govukelements " +
+  "is deprecated. From version 5.0, GOV.UK Frontend will not support compatibility " +
```

Comment:
@owenatgov I think I preferred your wording here? https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/settings/_warnings.scss#L10

**Current**
>…will not support compatibility with the legacy codebase govuk_elements

**Suggested**
>…will remove compatibility with legacy libraries such as govuk_elements

Reply:
Thanks Colin! I made a slight tweak to the suggestion so that each warning roughly reads:

>{variable} is deprecated. From version 5.0, GOV.UK Frontend will remove compatibility with the legacy library {library}.

I've done this because i'd rather keep each warning separate so that users are clearer on what in their codebase is kicking off the warning and why.